src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,69 @@
+export const AUTOCOMPLETE_LOADED = 'core/autocomplete/AUTOCOMPLETE_LOADED';
+export const AUTOCOMPLETE_STARTED = 'core/autocomplete/AUTOCOMPLETE_STARTED';
+export const AUTOCOMPLETE_ABORTED = 'core/autocomplete/AUTOCOMPLETE_ABORTED';
+
+const initialState = {
+  loading: false,
+  suggestions: [],
+};
+
+export function autocompleteStart({ errorHandlerId, filters }) {
+  if (!errorHandlerId) {
+    throw new Error('errorHandlerId is required');
+  }
+
+  if (!filters) {
+    throw new Error('filters are required');
+  }
+
+  return {
+    type: AUTOCOMPLETE_STARTED,
+    payload: { errorHandlerId, filters },
+  };
+}
+
+export function autocompleteAbort() {
+  return { type: AUTOCOMPLETE_ABORTED };
+}
+
+export function autocompleteLoad({ results }) {
+  if (!results) {
+    throw new Error('results are required');
+  }
+
+  return {
+    type: AUTOCOMPLETE_LOADED,
+    payload: { results },
+  };
+}
+
+export default function reducer(state = initialState, action = {}) {
+  const { payload } = action;
+
+  switch (action.type) {
+    case AUTOCOMPLETE_ABORTED:
```

Comment:
Roger that! I had `STOPPED` in mind first, but was not so nice. I did not think about `CANCELLED` but what you say here makes a lot of sense 💪